.github/workflows/readthedocs-builds.yml

```diff
@@ -0,0 +1,43 @@
+name: KFP Readthedocs Builds
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    paths:
+      - api/**
+      - sdk/**
+      - kubernetes_platform/**
+      - .github/workflows/readthedocs-builds.yml
+
+jobs:
+  test-readthedocs-builds:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+
+      - name: Install Dependencies
+        run: |
+          pip install -r docs/sdk/requirements.txt
+
+      - name: Build KFP SDK Docs
+        working-directory: docs/sdk
+        run: |
+          sphinx-build -b html . _build/html
+
+      - name: Build KFP Kubernetes SDK Docs
+        working-directory: kubernetes_platform/python/docs
+        run: |
+          sphinx-build -b html . _build/html
+
+      - name: Test K8s platform release script
```

Comment:
If this CI check is just to validate sphinx doc builds, I don't think this step is necessary/relevant.  If it is, we should rename the CI check to describe this as something related to 'Release Readiness' instead of simply doc builds.

Reply:
updated the ci workflow